apps/a11y-tests/tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": ["@uifabric/tslint-rules"],
+  "rules": {
+    "no-any": false,
```

Comment:
Please re-enable this and fix the issues (in places where you really need any, you can use `// tslint:disable-next-line:no-any`). Since this is new code, if at all possible we'd like it to be checked in with good practices.

Reply:
Removed `no-any` here and fixed TS breaks.